app/models/user.rb

```diff
@@ -101,6 +101,14 @@ def pending_profile
     profiles.gpo_verification_pending.order(created_at: :desc).first
   end
 
+  def fraud_review_pending?
+    @fraud_review_pending ||= fraud_review_pending_profile.present?
+  end
+
+  def fraud_review_pending_profile
+    profiles.where(fraud_review_pending: true).order(created_at: :desc).first
```

Comment:
It would make more sense to me to cache the profile in something like `@fraud_review_profile` rather than just cache whether it's present in `@fraud_review_pending`, if we need to cache it for performance reasons.